client/src/components/common/Header.js

```diff
@@ -0,0 +1,48 @@
+import React, { Component } from 'react';
+import { Menu } from 'semantic-ui-react';
+import PropTypes from 'prop-types';
+
+import './header.css';
+
+export default class Header extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      activeItem: 'home',
+    };
+    this.handleItemClick = this.handleItemClick.bind(this);
+  }
+
+  handleItemClick(e, { name }) {
```

Comment:
With this you don't need binding. Have a look at [this](https://hackernoon.com/the-constructor-is-dead-long-live-the-constructor-c10871bea599)
```suggestion
  handleItemClick = (e, { name }) => this.setState({ activeItem: name });
```

Reply:
Tried this, ESLINT wouldn't let me use this syntax @nimish-gupta

Reply:
What's the error it is throwing?

Reply:
![image](https://user-images.githubusercontent.com/10906358/53182151-ed51f180-361e-11e9-9411-7bb230fccced.png)
@nimish-gupta

Reply:
Check [this](https://github.com/babel/babel-eslint)

Reply:
Done :+1: